content/howto7/integration/implement-cicd-pipeline.md

```diff
@@ -39,7 +39,7 @@ To build the deployment package, use the [Team Server API](/apidocs-mxsdk/apidoc
 
 #### 3.2.1 Geting the Latest Revision
 
-To get the latest revision, use the [Retrieve Branch API call](/apidocs-mxsdk/apidocs/team-server-api#retrieve-branch) to get the `LatestRevisionNumber` from the output.
+To get the latest revision, use the [Retrieve Branch API call](/apidocs-mxsdk/apidocs/old-version/team-server-api#retrieve-branch) to get the `LatestRevisionNumber` from the output.
```

Comment:
This should use the new documentation. AFAIK, the old APIs will be deprecated once we make ours public.

Reply:
The link now points to the new APIs